app/angular/src/server/config/webpack.config.js

```diff
@@ -76,10 +76,6 @@ export default function(configDir) {
           loader: 'raw-loader',
           exclude: /\.async\.css$/,
         },
-        {
-          test: /\.scss$/,
```

Comment:
Don't we want to configure scss in case there is no angular-cli ?

Reply:
I think we should do this:
* `angular-cli` present -> use whatever's their config
* not present? Just fallback to good ol' css. 

Technically, if they're not using the cli we aren't officially supporting it. So they should config everything manually. This feels consistent.

@amcdnl what do you think?

Reply:
I think the vast majority of people will be using the cli so its pretty safe bet. There is the 1% case when they eject from the CLI we might need to think about ( https://github.com/angular/angular-cli/wiki/eject ) but that just nets the 'guts' of the CLI in a basic webpack config file as if you configured it manually yourself.

Personally, I'd fall under the thought process of try to handle as much as you can w/o config (seems to be a trend now i.e. parcel) so leaving the scss wouldn't hurt. Just my 2 cents tho.

Reply:
> so leaving the scss wouldn't hurt

Thanks your feedback, I'll revet this change 😄